frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -59,10 +73,12 @@ export const FeatureReleasePlanCard = ({
             <StyledIcon>
                 <Icon />
             </StyledIcon>
-            <div>
+            <StyledContentContainer>
                 <StyledName text={name} maxWidth='200' maxLength={25} />
-                <StyledDescription>{description}</StyledDescription>
-            </div>
+                <Tooltip title={description} arrow placement='top'>
```

Comment:
Adds a title line on top, with X button

Reply:
maybe I'm missing something, but why do we have a tooltip with the same text as in the "description"? What purpose does that serve?

Reply:
If it's for truncation purposes, isn't that exactly what the truncator does?